includes/service-providers/mailchimp/class-newspack-newsletters-mailchimp.php

```diff
@@ -1176,6 +1185,23 @@ function( $a, $b ) {
 					)
 				);
 			}
+
+			// Add tags if any were informed.
+			if ( ! empty( $contact['add_tags'] ) ) {
+				foreach ( $contact['add_tags'] as $tag ) {
+					$tag_id = $this->get_tag_id( $tag, true, $list_id );
+					$this->add_tag_to_contact( $email_address, $tag_id, $list_id );
+				}
+			}
+			// Remove tags if any were informed.
+			if ( ! empty( $contact['remove_tags'] ) ) {
```

Comment:
When I run the following commands:

```
$mc = Newspack_Newsletters_Mailchimp::instance();
$mc->add_contact( [ 'email' => 'raz+shell@automattic.com', 'name' => 'Test contact', 'add_tags' => ['tag-1','tag-2'] ], 'LISTID' );
$mc->add_contact( [ 'email' => 'raz+shell@automattic.com', 'name' => 'Test contact', 'remove_tags' => ['tag-2'] ], 'LISTID' );
```

The contact is created, however the `tag-2` is not removed in MC:
![Screenshot 2024-02-13 at 17 05 51](https://github.com/Automattic/newspack-newsletters/assets/17905991/a5e5be79-6c6d-474e-a916-2f331bf3fe7d)

Reply:
Aha! That did the trick 👍